CMakeLists.txt

```diff
@@ -1,47 +1,43 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
 project(pybind11_opencv)
 
 # Specify C++ Standard
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED on)
 
-find_package(catkin REQUIRED COMPONENTS
-  pybind11_catkin
-)
-
+find_package(ament_cmake REQUIRED)
+find_package(mpi_cmake_modules REQUIRED)
+find_package(pybind11 REQUIRED)
 find_package(OpenCV REQUIRED)
 
 
-catkin_package(
-  INCLUDE_DIRS include
-  CATKIN_DEPENDS pybind11_catkin
-  DEPENDS OpenCV
-)
-
 ###########
 ## Build ##
 ###########
 
-include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
-  ${OpenCV_INCLUDE_DIRS}
+add_pybind11_module(cvbind_test srcpy/cvbind_test.cpp
+    INCLUDE_DIRS ${OpenCV_INCLUDE_DIRS}
+    LINK_LIBRARIES ${OpenCV_LIBRARIES}
 )
 
```

Comment:
You should be creating a INTERFACE_LIBRARY:
`add_library(target_name INTERFACE)`
this way you can set dependencies to opencv directly here.
See:
http://mariobadr.com/creating-a-header-only-library-with-cmake.html

Reply:
Done.  Is it correct like this?

Reply:
Well because you actually do only a python binding here I guess this is fine.

For a header only library you can still create a target. I just wanted to make sure you know that.

I think this is good to be merged.